lib/matplotlib/widgets.py

```diff
@@ -1097,15 +1188,25 @@ def set_active(self, index):
         if index not in range(len(self.labels)):
             raise ValueError(f'Invalid CheckButton index: {index}')
 
-        cross_facecolors = self._crosses.get_facecolor()
-        cross_facecolors[index] = colors.to_rgba(
-            "black"
+        check_facecolors = self._checks.get_facecolor()
+        check_facecolors[index] = colors.to_rgba(
+            self._check_facecolors[index]
```

Comment:
I got very confused by `check_facecolors` and `_check_facecolors` here! If I understand this correctly, I think either
- renaming `self._check_facecolors` to `self._check_active_facecolors`
- renaming `check_facecolors` to `check_current_facecolors`
would help (and similarly below with edgecolors)

Reply:
This code doesn't even set the frame properties, so the `check_` prefix is redundant here, so I can drop it.